lib/ansible/galaxy/collection.py

```diff
@@ -55,6 +55,8 @@
 
 ModifiedContent = namedtuple('ModifiedContent', ['filename', 'expected', 'installed'])
 
+b_ignore_dirs = frozenset([b'CVS', b'.bzr', b'.hg', b'.git', b'.svn', b'__pycache__', b'.tox'])
+
```

Comment:
in any case, it should be in config with this as a default

Reply:
The configurable setting is additive to the hardcoded ignore list.